specification/network/resource-manager/Microsoft.Network/stable/2018-05-01/virtualNetworkGateway.json

```diff
@@ -2113,13 +2113,20 @@
     },
     "ConnectionSharedKey": {
       "properties": {
-        "value": {
-          "type": "string",
-          "description": "The virtual network connection shared key value."
-        }
+	    "properties": {
```

Comment:
are indents correct here?

Reply:
I think we don't need to change this part, but just adding the $ref below (lines 2123-2127 below). Updating the PR.